routes/exports.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+// Export Jobs
+$router->group([
+    'namespace' => 'Exports',
+    'middleware' => ['auth:api', 'scope:posts'],
+    'prefix' => '/exports/jobs'
+], function () use ($router) {
+    $router->get('/', 'JobsController@index');
+    $router->post('/', 'JobsController@store');
+    $router->get('/{id}', 'JobsController@show');
+    $router->put('/{id}', 'JobsController@update');
+    $router->delete('/{id}', 'JobsController@destroy');
```

Comment:
It's not clear why {id} is specified sometimes and at other times a regex to match a numeric, is there a difference in the id being passed?